clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -5118,10 +5118,6 @@ bool TokenAnnotator::mustBreakBefore(const AnnotatedLine &Line,
     return true;
   if (Left.IsUnterminatedLiteral)
     return true;
-  if (Right.is(tok::lessless) && Right.Next && Left.is(tok::string_literal) &&
```

Comment:
But why was that added? Alone from Github I can't navigate to the commit which introduced it.

Reply:
https://github.com/llvm/llvm-project/commit/2603ee0dc6003be0fbe43038b34580864d1e0c85
Here, 11 years ago

Reply:
Thanks.

And the test doesn't need to be changed?

Reply:
No, it still passes.